configuration/analyzer_config.json

```diff
@@ -772,6 +772,20 @@
       }
     }
   },
+  "Filescan": {
+    "type": "file",
+    "python_module": "filescan.FileScan",
+    "description": "Upload any file to scan for viruses in the content using File Scan API",
```

Comment:
`Extract IoCs from executable files, documents and scripts via [FileScan.io API](https://www.filescan.io/api/docs).`